addons/sale_coupon/models/sale_order.py

```diff
@@ -102,7 +102,7 @@ def _get_reward_values_product(self, program):
             'price_unit': - price_unit,
             'product_uom_qty': reward_qty,
             'is_reward_line': True,
-            'name': _("Free Product") + " - " + program.reward_product_id.name,
+            'name': _("Free Product - %s", program.reward_product_id.name),
```

Comment:
same here, only adding more complexity for the translators

Reply:
I'm maybe wrong, but in some languages, shouldn't the product name be placed before the "Free product" part of the text ? :thinking:

Reply:
The LTR is a rendering matter, you can display the same arabic text in LTR and RTL (doesn't mean anything but that's what you got if you are using a posbox for instance ^^).
اَلْعَرَبِيَّةُ
is rendered fine in my browser but copying this in sublimetext will switch the order (because it doesn't support RTL).
TL;DR placeholder position doesn't change in RTL languages.
Some languages like Japanese change the order of the words (verb position, etc.) but I don't think it is the case here as it's not really a sentence (but I may be wrong)

Reply:
There is also the case of punctuation/separator that might be different in some languages.

In this case it depends:
- if the `-` has a semantic meaning in the phrase (in which case it must be translated)
- or if it's just a design separator (in which case it probably should not be a single string in the first place, but 2 values that are not concatenated, and the views having to display them as 2 different information with a separator designed accordingly).

Honestly this seems like it just comes from us lazy (efficient?) developers that wanted to use existing framework possibilities to display the information as easily as possible instead of creating a more correct but more complicated way to pass the additional information.